Source/WebInspectorUI/UserInterface/Views/LogContentView.js

```diff
@@ -61,7 +61,7 @@ WI.LogContentView = class LogContentView extends WI.ContentView
         this._lastMessageView = null;
 
         this._findBanner = new WI.FindBanner(this, {alwaysShowing: true, className: "console"});
-        this._findBanner.visibilityPriority = WI.NavigationItem.VisibilityPriority.Low;
+        this._findBanner.visibilityPriority = WI.NavigationItem.VisibilityPriority.High;
```

Comment:
we probably should adjust this for some of the other navigation items to make sure that we have the right priorities (or really just change that entire system to not be number based)

but that can be a followup :)